cellpack/bin/pack.py

```diff
@@ -20,18 +23,29 @@
 ###############################################################################
 
 
-def pack(recipe, config_path=None, analysis_config_path=None):
+class DATABASE_IDS(Enum):
+    FIREBASE = "firebase"
+    GITHUB = "github"
```

Comment:
this should be in the db handler file or in `interface` which is where we store data objects/enums